apps/website/src/pages/docs/carousel.mdx

```diff
@@ -27,24 +25,22 @@ The currently shown slide can also be changed using the left/right arrow keys or
 
 ## Usage
 
+### Controlled State
+
 Carousels can be controlled externally by using the properties `activeSlideIndex` and `onSlideChange`.
 
 <LiveExample src='Carousel.controlled.tsx'>
   <AllExamples.CarouselControlledExample client:load />
 </LiveExample>
 
-The subcomponents can be used outside of Carousel. The following example shows how `Carousel.DotsList` can be used without the `Carousel` component. Clicking on the right half of the slide will advance the carousel to the next slide, whereas clicking on the left half will go to the previous slide.
+### Only Dots
+
+`Carousel.DotsList` can be used without the `Carousel` component. Clicking on the right half of the slide will advance the carousel to the next slide, whereas clicking on the left half will go to the previous slide.
 
 <LiveExample src='Carousel.onlyDots.tsx'>
   <AllExamples.CarouselOnlyDotsExample client:load />
 </LiveExample>
 
-## Props
```

Comment:
Since the PropsTable for each subcomponent is listed below the subcomponent and not at the end of the page, this change can be reverted.